### PR TITLE
Explain how link is used for BLOCK_CREATE

### PR DESCRIPTION
**Pull request by RickiNano**

### CODE REVIEW

File: docs/commands/rpc-protocol.md [hunk 1]
```diff
@@ -610,7 +610,7 @@ Parameters for state block:
 * `key` (optional): Instead of using "wallet" & "account" parameters, you can directly pass in a private key.
 * `source` (optional): The block hash of the source of funds for this receive block (the send block that this receive block will pocket).
 * `destination` (optional): The account that the sent funds should be accessible to.
```
Comment (zhyatt) [inline docs/commands/rpc-protocol.md:612]: @dsiganos Can you confirm if this optional `destination` for `block_create` RPC is supposed to be the account public key or the account address `nano_`? That is unclear and may need to be updated.

**Review by zhyatt — CHANGES_REQUESTED** [review]
Requesting some updates/details here. There is a related PR that is waiting on some clarity as well: https://github.com/nanocurrency/nano-docs/pull/561

File: docs/commands/rpc-protocol.md [hunk 2]
```diff
@@ -610,7 +610,7 @@ Parameters for state block:
...
-* `link` (optional): Instead of using "source" & "destination" parameters, you can directly pass "link" (source to receive or destination public key to send).
+* `link` (optional): Instead of using "source" & "destination" parameters, you can directly pass "link". If block is a send type, set link to the public key of the destination account. If it is a receive block, set link to the hash of the block to receive
```
[the remark this answers is not in the document]
Comment (RickiNano) [reply docs/commands/rpc-protocol.md]: Good suggestions. I have updated the pr

[the remark this answers is not in the document]
Comment (zhyatt) [reply docs/commands/rpc-protocol.md]: Thanks! Pushed a small link fix that was removed by GitHub in my comment above. Will wait for feedback on the other item before merging.

File: docs/commands/rpc-protocol.md (hunk 1 above)
Comment (zhyatt) [reply docs/commands/rpc-protocol.md]: Thanks @dsiganos I added links to the glossary to clarify this

**Pull request merged**